doc/scss/custom.scss

```diff
@@ -190,3 +190,29 @@ div.sk-text-image-grid-small {
 div.sk-text-image-grid-large {
   @include sk-text-image-grid(100px);
 }
+
+/* Customized Algolia Search Bar Style */
```

Comment:
@Charlie-XIAO I wanted to know if this is the right location to add customized CSS for the web page or I should have another CSS file somewhere?

Reply:
Yes this is the right place. `custom.scss` is for styles that applies to multiple pages and is added to every page.